### PR TITLE
Feature/red 117

### PR DESCRIPTION
**Pull request by red-tt-bear**

### CODE REVIEW

File: src/playground.ts [hunk 1]
```diff
@@ -12,10 +13,103 @@ if (configurations.email.enabled) {
 }
 
 import { sync } from './database';
+import schedulerHelper, { HttpMethod } from './utilities/scheduler-helper';
+
+const sleep = (millis: number): Promise<void> => new Promise(resolve => setTimeout(resolve, millis));
```
Comment (PanopticaRising) [inline src/playground.ts:18]: _immediately suspicious_

File: src/routes.ts [hunk 2]
```diff
@@ -44,4 +44,7 @@ router.use('/version', expressAsyncHandler(async (_req: express.Request, _res: e
         packageJson: version
     }));
 }));
+// TODO remove
```
Comment (PanopticaRising) [inline src/routes.ts:47]: Is this a dev route?

File: src/server.ts [hunk 3]
```diff
@@ -44,10 +44,36 @@ if (configurations.server.logAccess) {
...
+
 const limiter = rateLimit({
     windowMs: windowLength,
     max: maxRequests,
-    handler: (req, res, next) => next(Boom.tooManyRequests())
+    handler: (req, res, next) => next(Boom.tooManyRequests()),
+    skip: (req: Request): boolean => {
+        const { path: reqPath } = req;
+        if (!baseUrlRegex.test(reqPath)) {
+            logger.error(`A request a come in that did not meet the baseURL, this should not be possible! ${reqPath}`);
```
Comment (PanopticaRising) [inline src/server.ts:60]:
```suggestion
            logger.error(`A request came in that did not match the baseURL; this should not be possible!`, reqPath);
```

File: src/utilities/scheduler-helper.ts [hunk 4]
```diff
@@ -0,0 +1,157 @@
+import axios, { AxiosInstance, AxiosResponse } from 'axios';
+import * as path from 'path';
+
+const DEFAULT_SCHEDULER_URL = 'http://localhost:3003';
+
+const BASE_PATH = '';
+const SET_JOB_ENDPOINT = path.posix.join(BASE_PATH, '/set_job/');
```
Comment (PanopticaRising) [inline src/utilities/scheduler-helper.ts:7]: What's with the `posix`? Is that just specific cause we're running on Linux?

File: src/utilities/scheduler-helper.ts [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+    POST = 'POST',
+}
+
+export interface WebHookScheduleEvent {
+    // NOT IMPLEMENTED ON SCHEDULER SIDE
+    method: HttpMethod;
+    url: string;
+    // NOT IMPLEMENTED ON SCHEDULER SIDE
+    headers?: string;
+    data?: unknown;
```
Comment (PanopticaRising) [inline src/utilities/scheduler-helper.ts:29]: Should you hoist this above the others and have one NOT IMPLEMENTED AFTER comment?

**Review by PanopticaRising — APPROVED** [review]

File: src/playground.ts (hunk 1 above)
Comment (red-tt-bear) [reply src/playground.ts:18]: Haha it's just for the test, but yes you are correct!

**Pull request merged**